### ISSUE TITLE
SMT crashes on MultiSigWallet

### ISSUE THREAD

**Issue by axic** [ethereum/solidity#4566]
The file `test/compilationTests/MultiSigWallet/MultiSigWallet.sol` crashes with:
```
Internal compiler error during compilation:
/Users/alex/Projects/solidity/libsolidity/formal/SMTChecker.cpp(803): Throw in function smt::Expression dev::solidity::SMTChecker::currentValue(const dev::solidity::VariableDeclaration &)
Dynamic exception type: boost::exception_detail::clone_impl<dev::solidity::InternalCompilerError>
std::exception::what: 
[dev::tag_comment*] = 
```

**Comment by leonardoalt** [comment]
Missing support to modifier parameters

**Comment by leonardoalt** [comment]
The current bug is fixed by https://github.com/ethereum/solidity/pull/6244.
Then it still crashes because of compound assignment.

**Comment by leonardoalt** [comment]
@axic why did you close this? It still crashes

**Comment by axic** [comment]
Don't we have couple of issues tracking crashes due to lack of compound assignment?

**Comment by leonardoalt** [comment]
Yea but even when that's fixed this might still fail for some other reason. I think we should close this only when this examples is fully working.

**Comment by leonardoalt** [comment]
Fixed by #6006
